data/seeds/003_hospitals.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('hospitals').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('hospitals').insert([
+        {hospital_id: 1, hospital_name: 'Kirehe', amenities_id: "1", lat: -2.267769, long: 30.653169},
```

Comment:
I would change the ids to be 1, 2, 3 so we don't get any errors

Reply:
thank you for pointing that out